cli/jobs/nebulaml/bert-pretrain/job.yml

```diff
@@ -0,0 +1,23 @@
+# Training job submission via AML CLI v2
+
+$schema: https://azuremlschemas.azureedge.net/latest/commandJob.schema.json
+
+command: pip list && python pretrain_glue.py --save_steps 20 --deepspeed ds_config_bertbase.json --num_train_epochs 100 --output_dir outputs --disable_tqdm 1 --local_rank $RANK --evaluation_strategy "epoch" --logging_strategy "epoch" --per_device_train_batch_size 532 --gradient_accumulation_steps 1 --per_device_eval_batch_size 532 --learning_rate 3e-05 --adam_beta1 0.8 --adam_beta2 0.999 --weight_decay 3e-07 --warmup_steps 500 --fp16 --logging_steps 1000 --model_checkpoint "bert-base-uncased"
+
+experiment_name: bert-pretrain-nebula-ds-optimal
+environment: {your environment} # Should replace your environment
```

Comment:
Are you planning to add any scheduled validation of these workflows. 
If yes, how would these values be updated at the runtime with some valid values.
If not, how would be able to make sure the samples are in the valid/working state.

Reply:
users can check out the logs to validate, the guidance is in readme.